src/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Set the project name and version
+project(osconfig-fuzzer VERSION 1.0)
+
+# Specify the C++ standard
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Is this C++ version required for fuzzing?

Also, can we make this here conditional on clang so it won't execute for gcc?

Reply:
It already is conditional, see the src/CMakeLists.txt. C++ 17 is not required for fuzzing, but unless there is a need to go down, let's keep it reasonably fresh.

Reply:
Please see requirements for overall project:

set (CMAKE_C_STANDARD 11)
set (CMAKE_CXX_STANDARD 11)

If we need C++ 17 for fuzzing, let's isolate it so it won't be set unless building with clang (so we won't enter here when building with gcc and C++ 11)

Reply:
(even if the fuzzer code will be moved elsewhere in project, let's not depend on another condition and make a condition here, to say it explicit)

Reply:
I changed to C++ 11